### PR TITLE
Streamlit sidebar

### LINKED ISSUE #69: Add the last commit date and the last commit hash to the streamlit app's sidebar
To keep the information of the version we use

### PR DESCRIPTION
**Pull request by KarinDuong**
close #69

### CODE REVIEW

**Comment by pierrepo** [comment]
@KarinDuong could you share a screenshot?

**Comment by KarinDuong** [comment]
Capture d’écran du 2024-07-08 16-46-54

**Comment by pierrepo** [comment]
Thanks. Could you please avoid the bold font and output only the first 6 characters.

**Comment by KarinDuong** [comment]
6 charaters for both ? Because it will be `2024-0` and `116b9e`

**Comment by KarinDuong** [comment]
Or you want something like that : 
Capture d’écran du 2024-07-08 17-00-22

**Comment by pierrepo** [comment]
6 characters, only for the hash (we usually print the 6 first characters)

**Comment by pierrepo** [comment]
@KarinDuong are we ready to merge?

File: streamlit_app.py [hunk 1]
```diff
@@ -26,16 +26,20 @@
...
+        f"""
     **GroDecoder** extracts and identifies
     the molecular components of a structure file (PDB or GRO)
     issued from a molecular dynamics simulation.
 
     ---
-    [Source code](https://github.com/pierrepo/grodecoder)                 
+    [Source code](https://github.com/pierrepo/grodecoder)  
+
+    Last commit date: `{gd.get_git_last_commit_date()[1:-1]}`
```
Comment (pierrepo) [inline streamlit_app.py:37]: Why this slice?

Comment (pierrepo) [reply streamlit_app.py:37]: Why remove the first and last characters?

Comment (pierrepo) [reply streamlit_app.py:37]: I see. It's due to the extra single quotes returned by `get_git_last_commit_date()` and `get_git_last_commit_hash()` functions.
In these functions, could you replace `.strip()` by `strip("' \n")` ? It should solve the need for the weird slice later.

Comment (KarinDuong) [reply streamlit_app.py:37]: Done :white_check_mark:

**Pull request merged**